pypdf/filters.py

```diff
@@ -464,7 +461,7 @@ def decode(
         data: bytes,
         decode_parms: Optional[DictionaryObject] = None,
         **kwargs: Any,
-    ) -> str:
+    ) -> bytes:
```

Comment:
To which extent should we consider this a breaking change?

Reply:
this function is used internally there is no change.

Reply:
I disagree with this. `pypdf.filters` is a public module and thus possibly a breaking change, although there might not be many users of it.

Reply:
see above